packages/frontend-2/components/workspace/header/Header.vue

```diff
@@ -154,27 +164,41 @@ graphql(`
 
 enum ActionTypes {
   Settings = 'settings',
-  CopyLink = 'copy-link'
+  CopyLink = 'copy-link',
+  MoveProjects = 'move-projects',
+  Invite = 'invite'
 }
 
 const emit = defineEmits<{
   (e: 'show-invite-dialog'): void
   (e: 'show-settings-dialog', v: AvailableSettingsMenuKeys): void
+  (e: 'show-move-projects-dialog'): void
 }>()
 
 const props = defineProps<{
   workspaceInfo: WorkspaceHeader_WorkspaceFragment
 }>()
 
 const menuId = useId()
+const breakpoints = useBreakpoints(TailwindBreakpoints)
```

Comment:
JS breakpoints are never gonna be calculated correctly in SSR (since there's no screen), so preferably we want to rely on CSS instead. In this case, if you don't see any problems (e.g. SSR rendered page looking weird in mobile or desktop sizes) its probably fine.

Reply:
I cannot really rely on CSS, because it's used in a computed to conditionally add different actions to a context menu, tested it on mobile/desktop and different ways of loading but seems to be working as expected :)